Framework/DataHandling/src/SetSampleMaterial.cpp

```diff
@@ -74,6 +75,11 @@ void SetSampleMaterial::init() {
                   "Optional:  This total scattering cross-section (coherent + "
                   "incoherent) for the sample material in barns will be used "
                   "instead of tabulated");
+  const std::vector<std::string> extensions{".DAT"};
+  declareProperty(
```

Comment:
We unfortunately need to be careful with changes in the order of algorithm properties. The Python `simpleapi` functions auto generate their signatures and if we change the order then any calls in Python scripts that use only positional parameters will break. We have no other option than to put new things at the end.

Reply:
I had paused to consider this - the validations on this algorithm (and LoadSampleEnvironment) means they can't be called using positional arguments so I decided it was safe to inserted the new parameter in the logical place next to the absorption cross section. I don't think there can be any scripts out there that will be broken by this. What do you think?

Reply:
Yes I see what you are saying. It's not possible to form an ordered list of the parameters that is valid so they must have been specified by keyword. I agree this is fine.

Sorry for the confusion. I didn't think to check for `validateInputs` and made my usual comment about when I see changes like this.

Reply:
No problem, thanks for have a good look through